docs/meshcloud.payment-methods.md

```diff
@@ -0,0 +1,96 @@
+---
+id: meshstack.payment-methods
+title: Payment Methods
+---
+
+# Payment Methods
+
+## Introduction
```

Comment:
Because it is a common misconception we should mention in the introduction that payment methods are limited to one customer